components/custom-button.tsx

```diff
@@ -0,0 +1,13 @@
+import { blueDarkAlt } from '@pxblue/react-native-themes';
+import React from 'react';
+import { Button, useTheme } from 'react-native-paper';
+
+export const MyCustomButton: typeof Button = (props) => {
+    const theme = useTheme();
+    return (
+        <Button
+            {...props}
+            theme={props.mode === 'contained' && theme.dark ? Object.assign({}, blueDarkAlt, props.theme) : {}}
```

Comment:
This might help you with your theme mutation issue.
```
theme={props.mode === 'contained' && theme.dark ? Object.assign({...blueDarkAlt}, theme) : {}}
```

Reply:
These two comments can be applied to all the wrappers.